src/main/java/eu/xenit/gradle/alfrescosdk/internal/tasks/DefaultAmpSourceSetConfiguration.java

```diff
@@ -1,33 +1,46 @@
 package eu.xenit.gradle.alfrescosdk.internal.tasks;
 
-import eu.xenit.gradle.alfrescosdk.tasks.AmpSourceDirectories;
 import eu.xenit.gradle.alfrescosdk.tasks.AmpSourceSetConfiguration;
-import java.io.File;
-import java.util.Properties;
+import lombok.Getter;
 import org.gradle.api.Action;
 import org.gradle.api.Project;
+import org.gradle.api.file.SourceDirectorySet;
 import org.gradle.api.internal.file.SourceDirectorySetFactory;
 import org.gradle.api.provider.Property;
 import org.gradle.api.provider.Provider;
 import org.gradle.util.GUtil;
 
+import java.io.File;
+import java.util.Properties;
+
 public class DefaultAmpSourceSetConfiguration implements AmpSourceSetConfiguration {
 
-    private final DefaultAmpSourceDirectories config;
-    private final DefaultAmpSourceDirectories web;
+    @Getter private final SourceDirectorySet config;
+    @Getter private final SourceDirectorySet web;
     private final Project project;
-    private Property<Properties> moduleProperties;
-    private Property<Properties> fileMappingProperties;
+    @Getter private Property<Properties> moduleProperties;
+    @Getter private Property<Properties> fileMappingProperties;
     private Property<Boolean> dynamicExtension;
 
     public DefaultAmpSourceSetConfiguration(Project project, SourceDirectorySetFactory sourceDirectorySetFactory) {
```

Comment:
`sourceDirectorySetFactory` is no longer used.
This can be removed, I don't think it breaks any user-code ?

Reply:
I'm not sure. Have not tried without it.